packages/devtools_app/test/treemap_test.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/src/code_size/code_size_controller.dart';
+import 'package:devtools_app/src/code_size/code_size_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'support/wrappers.dart';
+
+void main() {
+  group('Treemap', () {
+    Future<void> pumpCodeSizeBody(
+      WidgetTester tester,
+      CodeSizeController controller,
+    ) async {
+      await tester.pumpWidget(wrapWithControllers(
+        const CodeSizeBody(),
```

Comment:
instead of pumping the code size screen, can you make this test pump the treemap widget directly?

Reply:
Done in #2116